src/app/my-messages/community-join-request-message-card/community-join-request-message-card.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit } from '@angular/core';
+import { TodoNotificationComponent } from '../todo-notification/todo-notification.component';
+import { CommunityRegistrationService } from '../../shared/community-registration.service';
+import { MatDialog } from '@angular/material';
+
+@Component({
+  selector: 'app-community-join-request-message-card',
+  templateUrl: './community-join-request-message-card.component.html',
+  styleUrls: ['./community-join-request-message-card.component.scss', '../my-messages.component.scss']
+})
+export class CommunityJoinRequestMessageCardComponent extends TodoNotificationComponent implements OnInit {
```

Comment:
The class `CommunityJoinRequestMessageCardComponent` implements `OnInit` interface but there is no `ngOnInit` implementation. I think `OnInit` interface can be removed from the hierarchy of the class.